docs/DesignDocs/SGX_QuoteEx_Integration.md

```diff
@@ -141,6 +141,10 @@ on the same platform.
     - If `SGX_AESM_ADDR` is defined (regardless of its value), out-of-process
     quote generation is done.
     - Otherwise if it is not defined, in-process quote generation is done.
+  - Since v0.15, another environment variable, `SGX_USE_IN_PROCESS_QUOTING`, controls the selection instead.
+    - If the value of `SGX_USE_IN_PROCESS_QUOTING` is set to 0, out-of-process quote generation is done.
+    - If the value of `SGX_USE_IN_PROCESS_QUOTING` is set to 1, in-process quote generation is done.
```

Comment:
Please make it clear that, both conditions have to be true to do in-process quote generation:
1. SGX_USE_IN_PROCESS_QUOTE=1
2. SGX_AESM_ADD is unset